validation-test/stdlib/Hashing.swift

```diff
@@ -72,54 +72,21 @@ HashingTestSuite.test("_mixInt/GoldenValues") {
 
 HashingTestSuite.test("_squeezeHashValue/Int") {
   // Check that the function can return values that cover the whole range.
-  func checkRange(_ r: Range<Int>) {
+  func checkRange(_ r: Int) {
     var results = [Int : Void]()
```

Comment:
Since we're cleaning up in here: shouldn't this just be a `Set<Int>`? (or if you're interested in perf, an Array<bool> initialized to `[false, false, false, ...]`)